projects/kubernetes-sigs/image-builder/build/build_image_on_metal.sh

```diff
@@ -139,4 +139,4 @@ OUTPUT_IMAGE_BLOB="*.gz"
 if [[ "$IMAGE_FORMAT" == "cloudstack" ]]; then
   OUTPUT_IMAGE_BLOB="*.qcow2"
 fi
-scp $SSH_OPTS $REMOTE_HOST:$REMOTE_HOME_DIR/$OUTPUT_IMAGE_BLOB $REPO_ROOT/$PROJECT_PATH/
+scp $SSH_OPTS $REMOTE_HOST:"$REMOTE_HOME_DIR/$OUTPUT_IMAGE_BLOB $ARTIFACTS_PATH/$IMAGE_FORMAT/$IMAGE_OS/EKSD_MANIFEST_URL $ARTIFACTS_PATH/$IMAGE_FORMAT/$IMAGE_OS/KUBERNETES_VERSION" $REPO_ROOT/$PROJECT_PATH/
```

Comment:
I believe these need to be on separate lines.

Reply:
Also `ARTIFACTS_PATH` has not been defined in this script, can you do that so this substitution works? You can set it at the top of the file along with other declarations and use it in the Line 130 as well.